llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskSchedulerService.java

```diff
@@ -1845,7 +1845,7 @@ private static boolean removeFromRunningTaskMap(TreeMap<Integer, TreeSet<TaskInf
    */
   private boolean shouldCycle(Map<String, List<NodeInfo>> availableHostMap) {
```

Comment:
could you please clarify here how changes in shouldCycle are related to the original patch (I mean the changes in selectHost) if related at all? (sorry I'm not really familiar with this part)